chatstripe/payment/models.py

```diff
@@ -4,15 +4,15 @@
 # Create your models here.
 
 class CardDetail(models.Model):
-    user = models.ForeignKey(User, on_delete=models.SET_NULL)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
SET_NULL is given because as a company the card details might be used in the future, we can't delete the card details.
So, SET_NULL basically soft deletes the cards, keep this as it was.